serverlessworkflow/modules/ROOT/pages/cloud/operator/build-and-deploy-workflows.adoc

```diff
@@ -131,6 +135,138 @@ spec:
+----
+
+[TIP]
+====
+Since the `buildArgs` attribute is an array of link:{kubernetes_envvar_url}[Kubernetes `EnvVar` object], you can refer to a `ConfigMap` or `Secret` to pass this value. You're not restricted by a plain value. On Minikube and Kubernetes, only `ConfigMap` and `Secret` are supported due to a restriction on the build system provided by these platforms.
+====
+
+The table below lists the Dockerfile arguments available in the default {operator_name} installation:
+
+.Default ARG definitions in the builder Dockerfile
+[cols="1,1,1"]
+|===
+| Argument | Description | Example
+
+|QUARKUS_EXTENSIONS | List of link:{quarkus_extensions_url}[Quarkus Extensions] separated by comma that the builder should add to the workflow. | org.kie.kogito:kogito-addons-quarkus-persistence-infinispan:2.0.0-SNAPSHOT
+|QUARKUS_ADD_EXTENSION_ARGS | Arguments passed to the Quarkus CLI when adding extensions. Enabled only when `QUARKUS_EXTENSIONS` is not empty. | See the link:{quarkus_cli_url}[Quarkus CLI documentation]
+|===
+
+=== Setting environment variables in the internal builder
+
+You can set environment variables to the `SonataFlowBuild` internal builder pod. 
+
+[IMPORTANT]
+====
+These environment variables are valid for the build context **only**. They are not set on the final built workflow image.
+====
+
+.Procedure
+
+1. Create or edit an existing `SonataFlowBuild` instance. It has the same name of the `SonataFlow` you're trying to build.
++
+.Checking if the SonataFlowBuild instance already exists
+[source,bash,subs="attributes+"]
+----
+# Check if the build exists
+kubectl get sfbuild/<name> -n <namespace>
+----
++
+2. Add to the `.spec.envs` the desired data.
++
+.Setting environment variables in the SonataFlowBuild instance
+[source,yaml,subs="attributes+"]
+----
+apiVersion: sonataflow.org/v1alpha08
+kind: SonataFlowBuild
+metadata:
+  name: <name>
+spec:
+  envs:
+    - name: MYENV1
+      value: value1
+    - name: MYENV2
+      value: value2
+----
++
+3. Save the file and apply it to the cluster. A new build should start soon with the new configuration.
+
+Alternatively, you can set this information in the `SonataFlowPlatform`, so that every new build instances will use it as a template. For example:
+
+.Example of a SonataFlowPlatform instance with default build args for every build within the namespace
+[source,yaml,subs="attributes+"]
+----
+apiVersion: sonataflow.org/v1alpha08
+kind: SonataFlowPlatform
+metadata:
+  name: <name>
+spec:
+  build:
+    template:
+      envs:
+        - name: MYENV1
+          value: value1
+        - name: MYENV2
+          value: value2
+----
+
+[TIP]
+====
+Since the `envs` attribute is an array of link:{kubernetes_envvar_url}[Kubernetes `EnvVar` object], you can refer to a `ConfigMap` or `Secret` to pass this value. You're not restricted by a plain value. On Minikube and Kubernetes, only `ConfigMap` and `Secret` are supported duo to a restriction on the build system provided by these platforms.
```

Comment:
```suggestion
Since the `envs` attribute is an array of link:{kubernetes_envvar_url}[Kubernetes `EnvVar` object], you can refer to a `ConfigMap` or `Secret` to pass this value. You're not restricted by a plain value. On Minikube and Kubernetes, only `ConfigMap` and `Secret` are supported due to a restriction on the build system provided by these platforms.
```